test-app/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "test-app",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Small description for test-app goes here",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "start": "ember serve",
+    "test": "ember test",
+    "test:ember": "ember test"
+  },
+  "dependenciesMeta": {
+    "@ember/test-helpers": {
+      "injected": true
+    }
+  },
+  "devDependencies": {
+    "@babel/eslint-parser": "^7.21.3",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
+    "@ember/jquery": "^2.0.0",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.0.1",
+    "@ember/test-helpers": "workspace:*",
+    "@embroider/test-setup": "^4.0.0",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
+    "concurrently": "^8.0.1",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli": "~4.12.2",
+    "ember-cli-app-version": "^6.0.0",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-htmlbars": "^6.2.0",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-data": "~4.11.3",
+    "ember-fetch": "^8.1.2",
+    "ember-load-initializers": "^2.1.2",
+    "ember-modifier": "^4.1.0",
+    "ember-page-title": "^7.0.0",
+    "ember-qunit": "^8.1.0",
+    "ember-resolver": "^10.0.0",
+    "ember-source": "~4.12.0",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^5.7.2",
+    "ember-try": "^3.0.0",
+    "eslint": "^8.37.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-ember": "^11.5.0",
+    "eslint-plugin-n": "^15.7.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-qunit": "^7.3.4",
+    "latest-version": "^5.0.0",
+    "loader.js": "^4.7.0",
+    "pretender": "^3.4.7",
+    "prettier": "^2.8.7",
+    "qunit": "^2.21.1",
+    "qunit-dom": "^3.2.0",
+    "tracked-built-ins": "^3.1.1",
+    "webpack": "^5.78.0"
+  },
+  "engines": {
+    "node": "14.* || 16.* || >= 18"
```

Comment:
wondering if we could drop 14 and 16..

Reply:
We could, but would me separate. During v2 conversion, node would cease to matter anyway